### PR TITLE
Ver 86416 better handling of vertica version

### PR DESCRIPTION
**Pull request by dmankins**
Address the bugs described in VER-86416 and VER-86417, which, were sort-of supposed-to-be documentation fixes, but as is often the case, fixing the documentation just makes the rough edges glaringly obvious.  So I sanded down some of the rough edges, too.  A bit beyond the scope of those two JIRAs, but I think a big improvement in usability.

The "you must run copy-template-to-sandbox by hand using this pile of bash" had been bothering me, so I fixed it.  Now it's just a command (setup-toolbox) that you invoke, which runs copy-template-to-sandbox for you.

vwasm-bash now gets built from a template by the Makefile; this lets us update the default Vertica version.

The fact that you have to run $WASMHOME/.env_setup every time you start vwasm-bash *also* bothered me, so I figured out how to fix that, too.  $WASMHOME/.env_setup is now an "init script", and vwasm-bash starts its bash in the container by running that init script.

### CODE REVIEW

File: Wasm-container/README.md [hunk 1]
```diff
@@ -92,43 +92,24 @@ $ make PACKAGE=vertica.deb VERTICA_VERSION=11.0.0-0
...
 # Using the container
 
-## Setting up your Web Assembly toolbox
+## Setting up your Web Assembly environment and toolbox
 
-To use this installation, before you run the container interactively, have it run the shell script `/usr/WebAssembly/tools/copy-template-to-sandbox` with an argument specifying the location of your WebAssembly toolbox (referred to as `$WASMHOME` in the commands illustrated below).
+### `setup-toolbox`
 
-The following set of commands illustrate how one might set up one's Wasm environment and then invoke the `/usr/WebAssembly/tools/copy-template-to-sandbox` script inside the container:
+To use this installation, before you run the container interactively, run set your `WASMHOME` environment variable to the name of a directory in your `$HOME` directory.  With this environment variable set, run `setup-toolbox` to copy the template Rust (`.cargo`) and Wasmer (`.wasmer`) direcotries into `$WASMHOME`.  Putting these into `$WASMHOME` instead of your home directory avoids interfering with any non-wasm Rust work you might be doing.  It also creates the `$HOME/WebAssembly/.env-setup` file, which is used by `vwasm-bash` as a startup profile.
```
Comment (cellio) [inline Wasm-container/README.md:99]: stray "run" in "run set your WASMHOME..."

Comment (cellio) [inline Wasm-container/README.md:99]: Does it really create $HOME/WebAssembly/.env_setup, or does it create $WASHMHOME/.env_setup?  Our example sets WASMHOME to $HOME/WebAssembly, but we can't assume people will always do that.  So let's be clear about where we're using env vars and when we're baking in specific paths.

File: Wasm-container/README.md [hunk 2]
```diff
@@ -92,43 +92,24 @@ $ make PACKAGE=vertica.deb VERTICA_VERSION=11.0.0-0
...
+### Launching an interactive shell to do Web Assembly development
 
 The shell-script `./vwasm-bash` starts an interactive shell inside the container.  
 
-When executing a shell inside the container, you need to set up your in-container PATH so the shell can find the `wasmer` and `rust` installations in the container.
-
-`wasmer` and rust installations are designed to be placed in a user's home directory.  This `copy-template-to-sandbox` command copied them to `$WASMHOME`.
+`./vwasm-bash` uses `$WASMHOME/.env-setup` to set up your in-container PATH so the shell can find the `wasmer` and `rust` installations in the container.
 
 To set up your environment appropriately, each time you start `vwasm-bash` run
```
Comment (cellio) [inline Wasm-container/README.md:114]: It sounds like this part isn't true any more? We're running .env-setup for you?

File: Wasm-container/README.md (hunk 1 above)
Comment (dmankins) [reply Wasm-container/README.md]: Quite right.  Fixed.

Comment (dmankins) [reply Wasm-container/README.md]: thanks, fixed.

File: Wasm-container/README.md (hunk 2 above)
Comment (dmankins) [reply Wasm-container/README.md]: Good catch. Fixed.

**Pull request merged**